scripts/mysqld_safe.sh

```diff
@@ -957,7 +974,7 @@ then
   fi
 
   # Launch mysqld with numactl.
-  cmd="$cmd numactl --interleave=all"
+  [ $simulate -eq 0 ] && cmd="$cmd numactl --interleave=all"
```

Comment:
The mariadb-service-convert is creating a config override file if the defaults have changed. I'd rather leave ExecStart omitted and hence refer to the default service file blank if numa_interleave isn't set.